DATA/MISSIONS/faction_prop.ini

```diff
@@ -561,52 +633,51 @@ legality = lawful
 nickname_plurality = singular
 msg_id_prefix = gcs_refer_faction_co_shi
 jump_preference = jumpgate
-npc_ship = npc_co_kt_fighter_d1-2
-npc_ship = npc_co_kt_fighter2_d1-2
-npc_ship = npc_co_kt_fighter3_d1-2
-npc_ship = npc_co_kt_fighter4_d1-2
-npc_ship = npc_co_kt_fighter_d3-4
-npc_ship = npc_co_kt_fighter2_d3-4
-npc_ship = npc_co_kt_fighter3_d3-4
-npc_ship = npc_co_kt_fighter4_d3-4
-npc_ship = npc_co_kt_fighter_d5-6
-npc_ship = npc_co_kt_fighter2_d5-6
-npc_ship = npc_co_kt_fighter3_d5-6
-npc_ship = npc_co_kt_fighter4_d5-6
+npc_ship = npc_co_kt_fighter_d1_a

```

Comment:
Why `npc_` where other loadouts do not have this `npc_` prefix?

Reply:
will be changed in future NPC Adjustment phases